### ISSUE TITLE
Does not work with 18.4, correct registry change has no effect

### ISSUE THREAD

**Issue by chucklepie** [nicholas-ross/SSMS-Schema-Folders#13]
Hello,
Test on SSMS 18.4 it does not work. First I think you need to change the registry key to this below (it is a different folder structure, but same guid, as other versions).

However, after applying this, you still get a loading fault asking to fix it in VS2017 and regardless, schema folders appears as an option in the settings dialog but is blank and you cannot use it due to the errors just mentioned.

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\SQL Server Management Studio\18.0_IsoShell_Config\Packages\{a88a775f-7c86-4a09-b5a6-890c4c38261b}]
"SkipLoading"=dword:00000001
```

**Comment by chucklepie** [comment]
My fault, it is working, looks like you do not need the registry entry.

The fix was as detailed in another issue, I had to unblock the DLLs. Maybe put this in the instructions?
